_includes/event-form.html

```diff
@@ -0,0 +1,137 @@
+<a
+  class="btn btn-round bg-purple text-white hover-white"
+  on="tap:event-lightbox"
+  role="button"
+  tabindex="-1"
+>
+  Join Us
+</a>
+
+<amp-lightbox
+  id="event-lightbox"
+  class="lightbox d-flex justify-content-center align-items-center"
+  layout="nodisplay"
+  scrollable
+  on="tap:event-lightbox.close"
+  tabindex="-1"
+  role="dialog"
+>
+  <form
+    class="col-6 col-md-6 col-sm-10 col-xs-10 px-11 pt-6 mt-6 mx-auto"
+    action-xhr="{{site.form_url}}/{{site.hubspot_portal_id}}/5346fc1c-3703-49e4-98bb-c35f5a055d0e"
```

Comment:
If this is different for austin and lubbock, make sure to abstract out the form guid in the configs like the other forms